LibTest/js_interop/JSExport/jsExport_A01_t07.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Annotation to allow Dart classes to be wrapped with a JS object
+/// using `dart:js_interop`'s `createJSInteropWrapper`.
+///
+/// When an instance of a class annotated with this annotation is passed to
+/// `createJSInteropWrapper`, the method returns a JS object that contains a
+/// property for each of the class' instance members. When called, these
+/// properties forward to the instance's corresponding members.
+///
+/// @description Checks that when an extension type is annotated with
+/// `@JSExport()` and passed to `createJSInteropWrapper`, the method returns a
+/// JS object that contains a property for each of the representation type's
+/// instance members.
+/// @author sgrekhov22@gmail.com
+
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
+import '../../../Utils/expect.dart';
+import '../js_utils.dart';
+
+String log = "";
+
+@JSExport()
+class C {
+  int variable = 42;
+  String method(String v) => "method($v);";
+  String get getter => "Some getter";
+  void set setter(bool value) {
+    log = "setter($value);";
+  }
+}
+
+@JSExport()
```

Comment:
We could expect a warning here because I don't see a good reason why we shouldn't.

It's a no-op already, so that leads me to believe that this would only add confusion if allowed.

Reply:
Very good! @sgrekhov, do we have a known and working way to detect these warnings? If we can do it then I'd recommend that the warning is expected and the test failure is approved when it's rolled, with a reference to an issue about implementing this warning. Similarly for the `extension` case below.

Reply:
I filled https://github.com/dart-lang/sdk/issues/61082 and updated the test to expect an `unspecified` error (this also covers warnings).

Reply:
OK, thanks!